package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "webpack-cordova-plugin",
-  "version": "0.1.5",
+  "name": "webpack-cordova-plugin-fork",
```

Comment:
Yes it is indeed! Before this PR was merged, I needed this to be in NPM but accidentally made my changes to the same branch as I used for this PR. Not sure how @markmarijnissen could publish new version to NPM.

Sorry for this, totally my mistake.